iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/ShiftedMarkedYamlEngineException.java

```diff
@@ -24,7 +24,18 @@
 import org.snakeyaml.engine.v2.exceptions.MarkedYamlEngineException;
 
 public class ShiftedMarkedYamlEngineException extends MarkedYamlEngineException {
+  private final Mark originalProblemMark;
+  private final Mark shiftedProblemMark;
+
   public ShiftedMarkedYamlEngineException(MarkedYamlEngineException original, Mark shiftedMark) {
     super(original.getContext(), original.getContextMark(), original.getProblem(), Optional.of(shiftedMark), original.getCause());
+    this.originalProblemMark = original.getProblemMark().orElse(shiftedMark);
+    this.shiftedProblemMark = shiftedMark;
+  }
+
+  public String describeShifting() {
+    // snakeyaml-engine is 0-based, so +1; shifted location comes from LocationShifter and is already adjusted
+    return "from [" + (originalProblemMark.getLine() + 1) + ":" + (originalProblemMark.getColumn() + 1) +
```

Comment:
For sure line number in our codebase is 1-based, but I'm not sure about the column.
We should double-check that.